src/wp-includes/class-wp-user-query.php

```diff
@@ -779,8 +775,12 @@ public function query() {
 			}
 
 			if ( isset( $qv['count_total'] ) && $qv['count_total'] ) {
+
 				/**
-				 * Filters SELECT FOUND_ROWS() query for the current WP_User_Query instance.
+				 * Return a total count of users.
+				 * Historically this ran SELECT FOUND_ROWS() after issuing the main query,
+				 * but since SQL_CALC_FOUND_ROWS is now deprecated from MySQL, it instead repeats
+				 * the same query with COUNT(*) instead of $this->query_fields.
```

Comment:
```suggestion
				 * Historically this ran `SELECT FOUND_ROWS()` after issuing the main query,
				 * but since `SQL_CALC_FOUND_ROWS` is now deprecated from MySQL, it instead repeats
				 * the same query with `COUNT(*)` instead of `$this->query_fields`.
```